ports/render96ex/render96ex.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+# PORTMASTER: sm64.zip, sm64.sh
+
+BASEROM='baserom.us.z64'
+RESTOOL_DIR="restool"
+RESTOOL_ZIP="restool.zip"
+
+# Ressources stuffs
+RES_DIR="res"
+BASEZIP="base.zip"
+DEMOS_DIR="demos"
+TEXTS_DIR="texts"
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+# We source the control.txt file contents here
+source $controlfolder/control.txt
+
+# With device_info we can get dynamic device information like resolution, cpu, cfw etc.
+source $controlfolder/device_info.txt
+
+# We source custom mod files from the portmaster folder example mod_jelos.txt which containts pipewire fixes
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+# We pull the controller configs like the correct SDL2 Gamecontrollerdb GUID from the get_controls function from the control.txt file here
+get_controls
+
+# We switch to the port's directory location below & set the variable for the gamedir and a configuration dir  easier handling below
+GAMEDIR="/$directory/ports/render96ex"
+CONFDIR="$GAMEDIR/conf/"
+
+# Ensure the conf directory exists
+mkdir -p "$CONFDIR"
+
+# Switch to the game directory
+cd $GAMEDIR
+
+# Log the execution of the script, the script overwrites itself on each launch
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+# Starting with the multiarch support we make sure that older PortMaster versions still revert to the old binary names. 
+export DEVICE_ARCH="${DEVICE_ARCH:-aarch64}"
+
+# Port specific additional libraries should be included within the port's directory in a separate subfolder named libs.aarch64, libs.armhf or libs.x64
+export LD_LIBRARY_PATH="${GAMEDIR}/libs.${DEVICE_ARCH}:${LD_LIBRARY_PATH}"
+
+# Add text_viewer binary path in PATH
+export PATH="${GAMEDIR}/bin.${DEVICE_ARCH}:${PATH}"
+
+# Provide appropriate controller configuration if it recognizes SDL controller input
+export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
+
+if [ -f "${controlfolder}/libgl_${CFW_NAME}.txt" ]; then
```

Comment:
this game does not use gl4es as it seems. So this whole libgl part is not needed.